.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true
```

Comment:
add this to the env
"node": true,
    "jest": true

Reply:
added